typescript/src/scss/main.scss

```diff
@@ -53,46 +52,41 @@
       }
     }
 
-
-    .grouped {
-      background-color: $low_opacity_gray;
-    }
-
-    /* Tooltip text */
     .tooltip{
       position: relative;
       padding: 1.115%;
+      .tooltiptext {
+        visibility: hidden;
+        width: 120px;
+        background-color: $dark_gray;
+        color: $white;
+        text-align: center;
+        padding: 5px 0;
+        border-radius: 6px;
+        position: absolute;
+        z-index: 1;
+        display: block;
+        bottom: 100%;
+      }
+      .tooltiptext::after {
+        content: " ";
+        position: absolute;
+        top: 100%; /* At the bottom of the tooltip */
+        left: 50%;
+        margin-left: -5px;
+        border-width: 5px;
+        border-style: solid;
+        border-color: $dark_gray transparent transparent transparent;
+      }
     }
 
-    .tooltip .tooltiptext {
-      visibility: hidden;
-      width: 120px;
-      background-color: $dark_gray;
-      color: $white;
-      text-align: center;
-      padding: 5px 0;
-      border-radius: 6px;
-      position: absolute;
-      z-index: 1;
-      display: block;
-      bottom: 100%;
-    }
-
-    /* Show the tooltip text when you mouse over the tooltip container */
     .tooltip:hover .tooltiptext {
       visibility: visible;
     }
```

Comment:
Can this be moved inside `.tooltip`? (You'll probably need to use a [parent selector](https://sass-lang.com/documentation/style-rules/parent-selector).)